.github/workflows/code-ql.yml

```diff
@@ -108,3 +108,27 @@ jobs:
 
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v3
+      with:
+        category: "/language:cpp"
+        output: sarif-results
+        upload: failure-only
+
+    - name: filter-sarif
+      uses: advanced-security/filter-sarif@v1
+      with:
+        patterns: |
+          -pktvisor/build/conan_home/**/*
+        input: sarif-results/cpp.sarif
+        output: sarif-results/cpp.sarif
+
+    - name: Upload SARIF
+      uses: github/codeql-action/upload-sarif@v3
```

Comment:
Is it immutable action?

Reply:
I think both are as they are maintained by github and not third party

Reply:
OK, let's try then!